src/sequences/SequenceFormula.ts

```diff
@@ -68,7 +70,7 @@ class SequenceFormula extends SequenceCached {
     }
 
     calculate(n: number) {
-        return this.formula.evaluate({n: n});
+        return BigInt(this.formula.evaluate({n: n}));
```

Comment:
Sorry, not sure what you mean. Are you saying this should be:
```
    return BigInt( this.formula.evaluate( { n: n } ) );
```
This one I have **very** strong feelings about: punctuation marks should be used as similarly in code as in natural language. I feel our eyes and minds are well trained to parse and understand punctuation that way. So since we put spaces before (but not immediately after) open parentheses (and the opposite for close parentheses), except in the case of immediately following punctuation, we should do that in code. And since we always put a space after a comma in prose, we should do that in code. Etc., etc. As a particular instance of this, I find what I wrote above
```
   return BigInt(this.formula.evaluate({n: n}));
```
far easier to read, and to tell that it is balanced, than the "spaced-out" version at the beginning of this comment. I just never see that anywhere else, and also not in most code that I have come across -- I have only seen a couple of projects that follow a "spaced-out" convention, and it drives me crazy. Of all of the various points of code formatting raised in this feedback, changing to "spaced-out" parentheses and braces would be by far the hardest for me to take. I strongly suggest you try simply following "Elements of Style" punctuation and spacing rules in code, and see if you don't find that it actually makes code easier to read, because it then looks like everything else you read.

The one place where I think the code author should have latitude is to sometimes write `x+1` and other times write `aLongExpression + anotherBigTerm` -- i.e. spaces surrounding arithmetic operators should be dictated by what makes that particular formula the most readable, not a strict rule.

Thoughts?

Reply:
I think I'll write up an issue where we can further discuss code style and formatting. I'll include a summary of comments from this PR.